homeassistant/components/media_player/plex.py

```diff
@@ -47,9 +49,23 @@
     cv.boolean,
 })
 
+PLEX_DATA = "plex"
+
+
+class PlexData:
+    """Storage class for platform global data."""
+
+    def __init__(self):
+        """Initialize the data."""
+        self.devices = {}
+        self.topology_lock = threading.Lock()
```

Comment:
You're not using this lock.

Reply:
I'd copied this from Sonos. Thought it was maybe needed for async or something to prevent parallel updates. (Read I didnt really understand what i was doing :) )